### PR TITLE
farmer test case

### PR DESCRIPTION
**Pull request by SumeetSanwal**
add product
list products

### CODE REVIEW

File: farmer/tests.py [hunk 1]
```diff
@@ -1,3 +1,39 @@
...
+
+    driver.get("http://127.0.0.1:8000/")
+
+    #login for customer
+    driver.find_element_by_id("your_name").send_keys("Kishan Kumar")
+    driver.find_element_by_id("your_pass").send_keys("kishan")
+    driver.find_element_by_id("signin_farmer").click()
+    time.sleep(1)
+    driver.find_element_by_id("signin").click()
+    time.sleep(2)
```
Comment (snigdha920) [inline farmer/tests.py:24]: what does this `time.sleep` do?

File: farmer/tests.py [hunk 2]
```diff
@@ -1,3 +1,39 @@
...
+
+    #Add new product
+    driver.find_element_by_xpath('//*[contains(concat( " ", @class, " " ), concat( " ", "panel-danger", " " ))]//*[contains(concat( " ", @class, " " ), concat( " ", "img-responsive", " " ))]').click()
+    driver.find_element_by_name('prodname').send_keys("Pumpkin")
+    driver.find_element_by_name('price').send_keys("100")
+
+    driver.find_element_by_name('image').send_keys(r"C:\Users\Sanwal\Desktop\pumpkin.jpg")
```
Comment (snigdha920) [inline farmer/tests.py:31]: What are we doing here? Can you point to me to link or something where I can understand what's happening?

Comment (SumeetSanwal) [reply farmer/tests.py:31]: This is how a farmer adds a new product but for that he needs to give name,price and upload and image so the last one is upload image for the product.

Comment (snigdha920) [reply farmer/tests.py:31]: oooh sexy

**Pull request merged**